src/Service.GraphQLBuilder/Mutations/MutationBuilder.cs

```diff
@@ -130,7 +129,10 @@ string returnEntityName
                 switch (operation)
                 {
                     case EntityActionOperation.Create:
-                        mutationFields.Add(CreateMutationBuilder.Build(name, inputs, objectTypeDefinitionNode, root, databaseType, entities, dbEntityName, returnEntityName, rolesAllowedForMutation));
+                        // Get the point/batch fields for the create mutation.
+                        Tuple<FieldDefinitionNode, FieldDefinitionNode> createMutationNodes = CreateMutationBuilder.Build(name, inputs, objectTypeDefinitionNode, root, databaseType, entities, dbEntityName, returnEntityName, rolesAllowedForMutation);
+                        mutationFields.Add(createMutationNodes.Item1); // Add field corresponding to point insertion.
+                        mutationFields.Add(createMutationNodes.Item2); // Add field corresponding to batch insertion.
```

Comment:
What if nested mutation is not enabled? Will we have Item2? Should this addition to mutationFields required?

Reply:
changed logic to return a list. Stale code.